src/crystal/system/unix/evented/event_loop.cr

```diff
@@ -38,13 +38,18 @@ module Crystal::Evented
   # allows optimizations to the OS (it can reuse already allocated resources),
   # and either the man page explicitly says so (Linux), or they don't (BSD) and
   # they must follow the POSIX definition.
-  protected class_getter arena = Arena(PollDescriptor).new(max_fds)
+  #
+  # The block size is set to 64KB because it's a multiple of:
+  # - 4KB (usual page size)
+  # - 1024 (common soft limit for open files)
+  # - sizeof(Arena::Entry(PollDescriptor))
```

Comment:
```suggestion
  # The block size is set to 64KB to hold 1000+ poll descriptor per block.
```

Reply:
But maybe we'd like a different block size?

Reply:
Perhaps we could make the size configurable at compile time?

Reply:
Through an ENV variable? Like `EVLOOP_ARENA_BLOCK_BYTESIZE`?

Reply:
That being said, I'm not sure it will have much impact: we grow once in a while when we reach N+ live file descriptors. It's currently over a 1000 live file descriptors.

Maybe we could use only 32KB for over 500 live fds. But we allocate 8MB for a lone fiber and 2×32KB for each IO (read/write), so I guess a static 64KB is negligible?

Reply:
For C10K, with 64KB we'd grow ~10 times, with 32KB it would be ~20 times.